main.js

```diff
@@ -0,0 +1,98 @@
+///////////////////////////////////////Variables
+
+var i = 0; i < 10;
+// var connect3 = (["", "", ""] || ["", "", ""] || ["", "", ""]);
+
+///////////////////////////////////////Game Controller
+
+function turnAlternator(){
+  changeTurn[i] = document.addEventListener('click', function(event){
```

Comment:
changeTurn is a function, not an array, so nothing is happening when you write [i] here.